wp-content/themes/pub/wporg-learn-2020/css/print.scss

```
@@ -0,0 +1,136 @@
+// Set custom margin
+@page {
+	margin: 1cm;
+}
+// Hide stuff that's not relevant on print media
+#wporg-header,
+#wporg-footer,
+#head-search,
+#wporg-header-menu,
+.site-header {
```

A couple other things to hide:

```suggestion
#wporg-header,
#wporg-footer,
#head-search,
#wporg-header-menu,
.site-header,
.workshop-page_video,
.wporg-learn-locale-notice {
```